### PR TITLE
mpv-unwrapped: build with meson

### PR DESCRIPTION
**Pull request by SuperSandro2000**
Currently breaks darwin.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Atemu** [comment]
Could you mark this as a draft until Darwin support is figured out?

Result of `nixpkgs-review pr 201379` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages marked as broken and skipped:</summary>
  <ul>
    <li>anki</li>
    <li>hydrus</li>
    <li>jellyfin-mpv-shim</li>
    <li>mnemosyne</li>
    <li>mpc-qt</li>
    <li>plex-mpv-shim</li>
    <li>sublime-music</li>
  </ul>
</details>
<details>
  <summary>8 packages failed to build:</summary>
  <ul>
    <li>ani-cli</li>
    <li>curseradio</li>
    <li>dmlive</li>
    <li>mpv-unwrapped</li>
    <li>python310Packages.mpv</li>
    <li>python39Packages.mpv</li>
    <li>somafm-cli</li>
    <li>ytfzf</li>
  </ul>
</details>


Build log:

```
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/0r3rnxkz71h7k7w30kjjq9q24z4vkx72-source
source root is source
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
patching script interpreter paths in version.py version.sh ./TOOLS/
version.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/x7mgi8y8r1aaww2q9xmh5pm3gvsynh0v-python3-3.10.8/bin/python3"
version.sh: interpreter directive changed from "#!/bin/sh" to "/nix/store/3hbyyikz32zlpkw2csqp1gry6qpniapw-bash-5.1-p16/bin/sh"
./TOOLS/umpv: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/x7mgi8y8r1aaww2q9xmh5pm3gvsynh0v-python3-3.10.8/bin/python3"
./TOOLS/appveyor-build.sh: interpreter directive changed from "#!/usr/bin/bash" to "/nix/store/3hbyyikz32zlpkw2csqp1gry6qpniapw-bash-5.1-p16/bin/bash"
./TOOLS/mpv_identify.sh: interpreter directive changed from "#!/bin/sh" to "/nix/store/3hbyyikz32zlpkw2csqp1gry6qpniapw-bash-5.1-p16/bin/sh"
./TOOLS/matroska.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/x7mgi8y8r1aaww2q9xmh5pm3gvsynh0v-python3-3.10.8/bin/python3"
./TOOLS/stats-conv.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/x7mgi8y8r1aaww2q9xmh5pm3gvsynh0v-python3-3.10.8/bin/python3"
./TOOLS/file2string.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/x7mgi8y8r1aaww2q9xmh5pm3gvsynh0v-python3-3.10.8/bin/python3"
./TOOLS/travis-rebuild-website: interpreter directive changed from "#!/bin/sh" to "/nix/store/3hbyyikz32zlpkw2csqp1gry6qpniapw-bash-5.1-p16/bin/sh"
./TOOLS/appveyor-install.sh: interpreter directive changed from "#!/usr/bin/bash" to "/nix/store/3hbyyikz32zlpkw2csqp1gry6qpniapw-bash-5.1-p16/bin/bash"
./TOOLS/dylib-unhell.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/x7mgi8y8r1aaww2q9xmh5pm3gvsynh0v-python3-3.10.8/bin/python3"
./TOOLS/macos-swift-lib-directory.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/x7mgi8y8r1aaww2q9xmh5pm3gvsynh0v-python3-3.10.8/bin/python3"
./TOOLS/macos-sdk-version.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/x7mgi8y8r1aaww2q9xmh5pm3gvsynh0v-python3-3.10.8/bin/python3"
./TOOLS/gen-osd-font.sh: interpreter directive changed from "#!/bin/sh" to "/nix/store/3hbyyikz32zlpkw2csqp1gry6qpniapw-bash-5.1-p16/bin/sh"
./TOOLS/osxbundle.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/x7mgi8y8r1aaww2q9xmh5pm3gvsynh0v-python3-3.10.8/bin/python3"
./TOOLS/idet.sh: interpreter directive changed from "#!/bin/sh" to "/nix/store/3hbyyikz32zlpkw2csqp1gry6qpniapw-bash-5.1-p16/bin/sh"
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
meson flags: --buildtype=plain         --libdir=/nix/store/h6iscgplg1idrbkvw2h7xgyps070mwqc-mpv-0.35.0/lib --libexecdir=/nix/store/h6iscgplg1idrbkvw2h7xgyps070mwqc-mpv-0.35.0/libexec         --bindir=/nix/store/h6iscgplg1idrbkvw2h7xgyps070mwqc-mpv-0.35.0/bin --sbindir=/nix/store/h6iscgplg1idrbkvw2h7xgyps070mwqc-mpv-0.35.0/sbin         --includedir=/nix/store/g25j99dz2dwb8b6rh26vlkggla64fqm8-mpv-0.35.0-dev/include         --mandir=/nix/store/r0iz6krymv8sm25d52sx2saypif65b5x-mpv-0.35.0-man/share/man --infodir=/nix/store/h6iscgplg1idrbkvw2h7xgyps070mwqc-mpv-0.35.0/share/info         --localedir=/nix/store/h6iscgplg1idrbkvw2h7xgyps070mwqc-mpv-0.35.0/share/locale         -Dauto_features=auto         -Dwrap_mode=nodownload         --prefix=/nix/store/h6iscgplg1idrbkvw2h7xgyps070mwqc-mpv-0.35.0 -Ddefault_library=shared -Dlibmpv=true -Dlibarchive=enabled -Dmanpage-build=enabled -Dcdda=disabled -Ddvbin=disabled -Ddvdnav=disabled -Dopenal=enabled -Dsdl2=enabled -Dmacos-cocoa-cb=disabled 
[1mThe Meson build system[0m
Version: 0.63.1
Source dir: [1m/private/tmp/nix-build-mpv-0.35.0.drv-0/source[0m
Build dir: [1m/private/tmp/nix-build-mpv-0.35.0.drv-0/source/build[0m
Build type: [1mnative build[0m
Project name: [1mmpv[0m
Project version: [1m0.35.0[0m
C compiler for the host machine: [1mclang[0m (clang 11.1.0 "clang version 11.1.0")
C linker for the host machine: [1mclang[0m ld64 609
Host machine cpu family: [1maarch64[0m
Host machine cpu: [1marm64[0m
Program [1mpython3[0m found: [1;32mYES[0m (/nix/store/x7mgi8y8r1aaww2q9xmh5pm3gvsynh0v-python3-3.10.8/bin/python3)
Found pkg-config: [1m/nix/store/5vbgc3y899ihzvs015n7b284pbfhys4x-pkg-config-wrapper-0.29.2/bin/pkg-config[0m (0.29.2)
Run-time dependency [1mlibavcodec[0m found: [1;32mYES[0m [36m58.134.100[0m
Run-time dependency [1mlibavfilter[0m found: [1;32mYES[0m [36m7.110.100[0m
Run-time dependency [1mlibavformat[0m found: [1;32mYES[0m [36m58.76.100[0m
Run-time dependency [1mlibavutil[0m found: [1;32mYES[0m [36m56.70.100[0m
Run-time dependency [1mlibswresample[0m found: [1;32mYES[0m [36m3.9.100[0m
Run-time dependency [1mlibswscale[0m found: [1;32mYES[0m [36m5.9.100[0m
Run-time dependency [1mlibass[0m found: [1;32mYES[0m [36m0.16.0[0m
Run-time dependency [1mthreads[0m found: [1;32mYES[0m
Compiler for C supports arguments -Werror=implicit-function-declaration: [1;32mYES[0m 
Compiler for C supports arguments -Wno-error=deprecated-declarations: [1;32mYES[0m 
Compiler for C supports arguments -Wno-error=unused-function: [1;32mYES[0m 
Compiler for C supports arguments -Wempty-body: [1;32mYES[0m 
Compiler for C supports arguments -Wdisabled-optimization: [1;32mYES[0m 
Compiler for C supports arguments -Wstrict-prototypes: [1;32mYES[0m 
Compiler for C supports arguments -Wno-format-zero-length: [1;32mYES[0m 
Compiler for C supports arguments -Wno-redundant-decls: [1;32mYES[0m 
Compiler for C supports arguments -Wvla: [1;32mYES[0m 
Compiler for C supports arguments -Wno-format-truncation: [1;31mNO[0m 
Compiler for C supports arguments -Wimplicit-fallthrough: [1;32mYES[0m 
Compiler for C supports arguments -fno-math-errno: [1;32mYES[0m 
Compiler for C supports arguments -Wformat -Werror=format-security: [1;32mYES[0m 
Compiler for C supports link arguments -Wl,-z,noexecstack: [1;31mNO[0m 
Compiler for C supports link arguments -Wl,--nxcompat,--no-seh,--dynamicbase: [1;31mNO[0m 
Library [1mdl[0m found: [1;32mYES[0m
Checking for function "[1mdlopen[0m" with dependency -ldl: [1;32mYES[0m 
Compiler for C supports link arguments -rdynamic: [1;32mYES[0m 
Run-time dependency [1mappleframeworks[0m found: [1;32mYES[0m (Cocoa, IOKit, QuartzCore)
Library [1mwindowsapp[0m skipped: feature [1muwp[0m disabled
Checking for function "[1mglob[0m" : [1;32mYES[0m 
Header "[1msys/vt.h[0m" has symbol "[1mVT_GETMODE[0m" : [1;31mNO[0m 
Header "[1msys/consio.h[0m" has symbol "[1mVT_GETMODE[0m" : [1;31mNO[0m 
Checking if "[1mglibc-thread-name check[0m" compiles: [1;31mNO[0m 
Checking if "[1mosx-thread-name check[0m" compiles: [1;32mYES[0m 
Checking for function "[1mfstatfs[0m" : [1;32mYES[0m 
Checking for function "[1mfstatfs[0m" : [1;31mNO[0m 
Checking if "[1mvector check[0m" compiles: [1;32mYES[0m 
Program [1m/private/tmp/nix-build-mpv-0.35.0.drv-0/source/TOOLS/file2string.py[0m found: [1;32mYES[0m (/private/tmp/nix-build-mpv-0.35.0.drv-0/source/TOOLS/file2string.py)
Program [1m/private/tmp/nix-build-mpv-0.35.0.drv-0/source/TOOLS/matroska.py[0m found: [1;32mYES[0m (/private/tmp/nix-build-mpv-0.35.0.drv-0/source/TOOLS/matroska.py)
Program [1m/private/tmp/nix-build-mpv-0.35.0.drv-0/source/version.py[0m found: [1;32mYES[0m (/private/tmp/nix-build-mpv-0.35.0.drv-0/source/version.py)
Dependency [1mlibcdio_paranoia[0m skipped: feature [1mcdda[0m disabled
Dependency [1mdvdnav[0m skipped: feature [1mdvdnav[0m disabled
Dependency [1mdvdread[0m skipped: feature [1mdvdnav[0m disabled
Run-time dependency [1miconv[0m found: [1;32mYES[0m
Run-time dependency [1mmujs[0m found: [1;32mYES[0m [36m1.2.0[0m
Run-time dependency [1mlcms2[0m found: [1;32mYES[0m [36m2.13[0m
Run-time dependency [1mlibarchive[0m found: [1;32mYES[0m [36m3.6.1[0m
Run-time dependency [1mlibavdevice[0m found: [1;32mYES[0m [36m58.13.100[0m
Run-time dependency [1mlibbluray[0m found: [1;32mYES[0m [36m1.3.2[0m
Library [1mm[0m found: [1;32mYES[0m
Library [1mrt[0m found: [1;31mNO[0m
Run-time dependency [1mlua[0m found: [1;32mYES[0m [36m5.2.4[0m
Run-time dependency [1mrubberband[0m found: [1;32mYES[0m [36m3.1.0[0m
Run-time dependency [1msdl2[0m found: [1;32mYES[0m [36m2.24.2[0m
Library [1matomic[0m found: [1;31mNO[0m
Run-time dependency [1muchardet[0m found: [1;32mYES[0m [36m0.0.7[0m
Did not find CMake 'cmake'
Found CMake: [1;31mNO[0m
Run-time dependency [1mvapoursynth[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mvapoursynth-script[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mzimg[0m found: [1;32mYES[0m [36m3.0.4[0m
Run-time dependency [1mzlib[0m found: [1;32mYES[0m [36m1.2.13[0m
Run-time dependency [1malsa[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mappleframeworks[0m found: [1;32mYES[0m (Foundation, AudioToolbox)
Header "[1mAudioToolbox/AudioToolbox.h[0m" has symbol "[1mkAudioUnitSubType_RemoteIO[0m" : [1;31mNO[0m 
Run-time dependency [1mappleframeworks[0m found: [1;32mYES[0m (CoreFoundation, CoreAudio, AudioUnit, AudioToolbox)
Run-time dependency [1mjack[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mopenal[0m found: [1;32mYES[0m [36m1.22.2[0m
Library [1mOpenSLES[0m found: [1;31mNO[0m
Header "[1msys/soundcard.h[0m" has symbol "[1mSNDCTL_DSP_SETPLAYVOL[0m" : [1;31mNO[0m 
Run-time dependency [1mlibpipewire-0.3[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mlibpulse[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1msndio[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Header "[1maudioclient.h[0m" has symbol "[1mIAudioClient[0m" : [1;31mNO[0m 
Run-time dependency [1mcaca[0m found: [1;32mYES[0m [36m0.99.beta20[0m
Check usable header "[1md3d9.h[0m" skipped: feature [1mdirect3d[0m disabled
Run-time dependency [1mlibdrm[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mgbm[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mlibjpeg[0m found: [1;32mYES[0m [36m2.1.4[0m
Run-time dependency [1mlibplacebo[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mshaderc[0m found: [1;31mNO[0m (tried pkgconfig, pkgconfig, pkgconfig and system)
Run-time dependency [1mlibsixel[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mspirv-cross-c-shared[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mwayland-client[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mwayland-cursor[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mwayland-protocols[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mxkbcommon[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Check usable header "[1mlinux/input-event-codes.h[0m" : [1;31mNO[0m 
Program [1mwayland-scanner[0m found: [1;31mNO[0m
Run-time dependency [1mx11[0m found: [1;32mYES[0m [36m1.8.1[0m
Run-time dependency [1mxscrnsaver[0m found: [1;32mYES[0m [36m1.2.3[0m
Run-time dependency [1mxext[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mxinerama[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mxpresent[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mxrandr[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Dependency [1mxv[0m skipped: feature [1mxv[0m disabled
Run-time dependency [1mappleframeworks[0m found: [1;32mYES[0m (OpenGL)
Run-time dependency [1megl[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Library [1mEGL[0m skipped: feature [1megl-android[0m disabled
Run-time dependency [1mwayland-egl[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Dependency [1m/opt/vc/lib/pkgconfig/brcmegl.pc[0m skipped: feature [1mrpi[0m disabled
Dependency [1mvulkan[0m skipped: feature [1mvulkan[0m disabled
Run-time dependency [1mffnvcodec[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Header "[1mOpenGLES/ES3/glext.h[0m" has symbol "[1mGL_RGB32F[0m" : [1;31mNO[0m 
Dependency [1m/opt/vc/lib/pkgconfig/mmal.pc[0m skipped: feature [1mrpi-mmal[0m disabled
Run-time dependency [1mlibva[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mlibva-drm[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mlibva-wayland[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Run-time dependency [1mlibva-x11[0m found: [1;31mNO[0m (tried pkgconfig, framework and cmake)
Dependency [1mvdpau[0m skipped: feature [1mvdpau[0m disabled
Program [1m/private/tmp/nix-build-mpv-0.35.0.drv-0/source/TOOLS/macos-sdk-version.py[0m found: [1;32mYES[0m (/private/tmp/nix-build-mpv-0.35.0.drv-0/source/TOOLS/macos-sdk-version.py)
Program [1mxcrun[0m found: [1;31mNO[0m
Checking if "[1mmacos-touchbar check[0m" compiles: [1;32mYES[0m 
Program [1mrst2man[0m found: [1;32mYES[0m (/nix/store/yps67ivv0zvfxpbbslf9r1bs9iv157cv-python3.10-docutils-0.19/bin/rst2man)
Program [1mrst2html[0m skipped: feature [1mhtml-build[0m disabled
Program [1mrst2pdf[0m skipped: feature [1mpdf-build[0m disabled
Configuring [1mconfig.h[0m using configuration
[1mMessage:[0m List of enabled features: bsd-fstatfs build-date caca cocoa coreaudio cplayer cplugins ffmpeg gl gl-cocoa glob glob-posix gpl iconv javascript jpeg lcms2 libarchive libass libavdevice libbluray libdl libm libmpv-shared lua macos-touchbar manpage-build openal osx-thread-name plain-gl posix rubberband rubberband-3 sdl2 sdl2-audio sdl2-gamepad sdl2-video threads uchardet vector videotoolbox-gl zimg zlib
Fetching value of define "[1mMPV_CLIENT_API_VERSION[0m" : (((2) << 16) | (0) | 0UL) 

meson.build:1660:4: [1;31mERROR:[0m No host machine compiler for 'osdep/macosx_application.m'

A full log can be found at [1m/private/tmp/nix-build-mpv-0.35.0.drv-0/source/build/meson-logs/meson-log.txt[0m
```

[meson-log.txt](https://github.com/NixOS/nixpkgs/files/10019501/meson-log.txt)

**Comment by Atemu** [comment]
It seems to be unable to find our SDK. If you enable `macos-10-14-features` (which our SDK certainly supports) you get:

```
meson.build:1470:0: ERROR: Feature macos-10-14-features cannot be enabled: a suitable macos sdk version could not be found!
```

**Comment by Atemu** [comment]
OH. That's our doing. Trivial to patch. Was wondering why I couldn't find it in their meson.build.

**Comment by Atemu** [comment]
Apparently MPV includes an osxbundle target or something now, we should probably use that instead of our own install script for Darwin?

File: pkgs/applications/video/mpv/default.nix [hunk 1]
```diff
@@ -100,37 +99,31 @@ in stdenv.mkDerivation rec {
   NIX_LDFLAGS = lib.optionalString x11Support "-lX11 -lXext "
     + lib.optionalString stdenv.isDarwin "-framework CoreFoundation";
```
Comment (Atemu) [inline pkgs/applications/video/mpv/default.nix:100]: We don't need the LDFLAGS for darwin here anymore, we should check whether we can drop this for Linux aswell.

**Comment by Atemu** [comment]
Got it working: https://github.com/Atemu/nixpkgs/tree/mpv-meson-darwin

**Comment by Atemu** [comment]
@SuperSandro2000 do you want to continue to pursue this?

**Comment by SuperSandro2000** [comment]
Pulled, squashed, edited message and Co-Author, try to test but I need a real X11 for that and force pushed.

**Comment by SuperSandro2000** [comment]
Do we want to merge this?

**Review by AndersonTorres — APPROVED** [review]

**Review by Philipp-M — APPROVED** [review]
Works for me on NixOS.

I would suggest changing `ffmpeg` to `ffmpeg_5` in this PR to avoid issues like these: 
https://github.com/mpv-player/mpv/issues/7326

**Review by Atemu — APPROVED** [review]
Result of `nixpkgs-review pr 201379` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages marked as broken and skipped:</summary>
  <ul>
    <li>anki</li>
    <li>hydrus</li>
    <li>jellyfin-mpv-shim</li>
    <li>mnemosyne</li>
    <li>mpc-qt</li>
    <li>plex-mpv-shim</li>
    <li>sublime-music</li>
  </ul>
</details>
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>ani-cli</li>
    <li>curseradio</li>
    <li>dmlive</li>
    <li>mpv-unwrapped</li>
    <li>python310Packages.mpv</li>
    <li>python39Packages.mpv</li>
    <li>somafm-cli</li>
    <li>ytfzf</li>
  </ul>
</details>

@AndersonTorres Now that branch-off has happened, Darwin support is figured out and it's been sitting around for a while with no obvious flaws pointed out, there's nothing to "go to war" about.

**Pull request merged**